src/app/admin/modals/user-notification-settings-modal/notification.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="modal-dialog">
+  <div class="modal-content">
+    <form [formGroup]="notificationSettings" (ngSubmit)="saveNotifications()" novalidate>
+      <div class="modal-header">
+        <h3>Notification Settings</h3>
+      </div>
+      <div class="modal-body">
+        <div class="form-group">
```

Comment:
`form-group` is from [bootstrap form](https://getbootstrap.com/docs/4.0/components/forms/), should change to Material

Reply:
Thanks! I'll have a look at it.